### DISCUSSION TITLE
Android inegration/app

### DISCUSSION THREAD

**Discussion by goodcoderfromeurope** [web-scrobbler/web-scrobbler#3333] [Q&A]
Hi,
Aby idea how to integrate this with android?
Cheers

**Comment by inverse** [comment]
if its just for native media players https://github.com/kawaiiDango/pScrobbler

but for browsers, perhaps itll work as a firefox extension?

**Reply by jonpsp** [reply]
I use Pano Scrobbler on my phone - it picks up track details when I play music in my browsers no matter what the website is (Brave, Firefox), my music apps (YouTube, YouTube Music, Spotify), and even works with Shazam (if Auto Shazam is active). Pano Scrobbler can also listen to what is being played to work out what it is, similar to Shazam or Sound Hound, but uses a different database so can identify songs that the others miss.
